config/cobbler/distro_signatures.json

```diff
@@ -673,38 +673,7 @@
           ]
         }
       },
-      "fedora32": {
```

Comment:
Why remove fedora32?

Reply:
Fixed. Thanks for the review. Copy Pasta Error...